pkg/controller/node/nodecontroller.go

```diff
@@ -215,7 +215,7 @@ func NewNodeController(
 	runTaintManager bool,
 	useTaintBasedEvictions bool) (*NodeController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "controllermanager"})
+	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "node-controller"})
```

Comment:
I simply search `EventSource` in pkg/controller and find that there are only four places using `controllermanager` and other places using more accurate names such as [job-controller](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/job/jobcontroller.go#L92), [persistentvolume-controller](https://github.com/knightXun/kubernetes/blob/b939199fcf05584624ae734b57122c5a17539751/pkg/controller/volume/persistentvolume/pv_controller_base.go#L75) :)